scripts/test-api-endpoint.ts

```
@@ -0,0 +1,68 @@
+#!/usr/bin/env npx tsx
+import * as dotenv from 'dotenv'
+import * as path from 'path'
+
+// Load .env.local explicitly
+dotenv.config({ path: path.resolve(process.cwd(), '.env.local') })
+
+// Set up the environment variable for Cloudflare KV to be used
+process.env.CLOUDFLARE_KV_NAMESPACE_ID = process.env.CLOUDFLARE_KV_NAMESPACE_ID
```

Biome (1.9.4)

[error] CLOUDFLARE_KV_NAMESPACE_ID is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)